src/main/java/com/coopetico/coopeticobackend/controladores/TaxistasControlador.java

```diff
@@ -43,23 +33,37 @@
     /**
      * Servicio de taxistas para consultar datos.
      */
-    @Autowired
-    private TaxistasServicio taxistaServicio;
+    private final TaxistasServicio taxistaServicio;
 
     /**
      * Servicio generador de token para establecer la contrasenna.
      */
-    @Autowired
-    private TokensRecuperacionContrasenaServicioImpl tokensServicio;
+    private final TokensRecuperacionContrasenaServicioImpl tokensServicio;
 
     /**
      * Servicio para enviar correos.
      */
-    @Autowired
-    private EmailServiceImpl correoServicio;
+    private final EmailServiceImpl correoServicio;
 
+    /**
+     * Logger para subir la imagen
+     */
+    private final Logger log = LoggerFactory.getLogger(TaxistasControlador.class);
 
-    private final Logger log = LoggerFactory.getLogger(UsuarioControlador.class);
+    /**
+     * Utilidades para subir y eliminar imagenes, para no duplicar tanto código
+     */
+    private UtilidadesControlador utilidadesControlador;
+
+    /**
+     * Constructor
+     */
+    private TaxistasControlador(TaxistasServicio taxistaServicio, TokensRecuperacionContrasenaServicioImpl tokensServicio, EmailServiceImpl correoServicio){
```

Comment:
Si utiliza la notación @Autowired, este constuctor lo único que debería tener es 
constructor() {
 this.utilidadesControlador = new UtilidadesControlador();
}